pftools/python/parflow/tools/ref/well_keys.yaml

```diff
@@ -0,0 +1,97 @@
+# -----------------------------------------------------------------------------
```

Comment:
This file defines all the fields the builder may expect to encounter. if you try to enter a field into excel that is not in here you will get an error